### PR TITLE
feat(shim-mgmt): iptables handler

### PR DESCRIPTION
**Pull request by Ji-Xinyou**
Support kata-ctl iptables series of commands in runtime.

Fixes: #5370
Signed-off-by: Ji-Xinyou <jerryji0414@outlook.com>

### CODE REVIEW

File: src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs [hunk 1]
```diff
@@ -269,6 +271,28 @@ impl Sandbox for VirtSandbox {
...
     }
+
+    async fn set_iptables(&self, is_ipv6: bool, data: Vec<u8>) -> Result<Vec<u8>> {
+        info!(sl!(), "sb: set_iptables invoked");
+        let req = SetIPTablesRequest { is_ipv6, data };
+        let resp = self
+            .agent
+            .set_ip_tables(req)
+            .await
+            .context("sandbox: failed to set iptable")?;
```
Comment (liubin) [inline src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs:282]: nit: `iptable` -> `iptables`

File: src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs [hunk 2]
```diff
@@ -269,6 +271,28 @@ impl Sandbox for VirtSandbox {
...
+    }
+
+    async fn get_iptables(&self, is_ipv6: bool) -> Result<Vec<u8>> {
+        info!(sl!(), "sb: get_iptables invoked");
+        let req = GetIPTablesRequest { is_ipv6 };
+        let resp = self
+            .agent
+            .get_ip_tables(req)
+            .await
+            .context("sandbox: failed to get iptable")?;
```
Comment (liubin) [inline src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs:293]: nit: `iptable` -> `iptables`

**Review by liubin — APPROVED** [review]

**Review by Tim-0731-Hzt — APPROVED** [review]

File: src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs (hunk 1 above)
Comment (Ji-Xinyou) [reply src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs]: Fixed

File: src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs (hunk 2 above)
Comment (Ji-Xinyou) [reply src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs]: Fixed

**Pull request merged**